nautobot/users/factory.py

```diff
@@ -31,3 +46,16 @@ class Meta:
     last_login = factory.Faker("date_time", tzinfo=timezone.utc)
 
     # TODO config_data
+
+
+class SavedViewFactory(BaseModelFactory):
+    class Meta:
+        model = SavedView
+
+    name = factory.LazyAttributeSequence(lambda o, n: f"Sample {o.view} Saved View - {n + 1}")
+    owner = random_instance(User, allow_null=False)
+    view = factory.Faker(
+        "random_element",
+        elements=VIEW_NAMES,
+    )
+    # TODO config attribute
```

Comment:
Probably would be sufficient (and exercise some data validation checks) in the short term to just use the random-dict factory.